src/components/toolbox/infoBanner/infoBanner.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import { withTranslation } from 'react-i18next';
+// import { useSelector } from 'react-redux';
```

Comment:
@gadjacobs Please remove the comments here

Reply:
Thanks. I commented it in testing to be able to test without having to login

Reply:
If so, then remember to update them before the PR gets merged